client/cmd/replay/replay.go

```diff
@@ -16,6 +16,7 @@ func NewReplayCommand() *cobra.Command {
 
 	cmd.AddCommand(
 		CreateCommand(),
+		ListCommand(),
```

Comment:
I believe there should be another command, the get/status command, given a replay ID. We can show the status of each of the runs for a single Replay, and can also show the configs that cannot be shown in the list command, for example, the job's config and the sequential/parallel mode.

However, we can also raise another PR for this.

Reply:
yes, agree. We can provide another PR for detailed replay ID status. List command only shows the essential not detailed one

Reply:
here's the PR: https://github.com/goto/optimus/pull/11